### PR TITLE
PreAuth Clarification

### PR DESCRIPTION
**Pull request by thecapacity**
Suggestion on language to clarify that "pre-auth" status is itself a form of Authorization to Operate.

Assumes "Zero ATO" works as a valid short-name, as an analogue (based on CC0 associations)

### CODE REVIEW

**Comment by thecapacity** [comment]
Based on a discussion with @NoahKunin - https://18f.slack.com/archives/compliance/p1470922051000188

**Comment by NoahKunin** [comment]
Leaving it open for 36 hours for other thoughts.

File: _pages/ato/types.md [hunk 1]
```diff
@@ -3,15 +3,16 @@ title: Types of ATO
...
 
+* [Pre-authorization for internal government use](#pre-authorization)
 * [90-day Lightweight Authority to Operate (LATO)](#day-authorization)
 * [1-year Lightweight Authority to Operate (LATO)](#year-authorization)
 * [Continuous Authority to Operate (cATO)](#continuous-authorization)
 
 ### Pre-authorization
 
-You may operate without authorization if all of the following conditions are met:
+You may operate without further authorization, based on our approved pre-existing security authorization, if all of the following conditions are met:
```
Comment (afeld) [inline _pages/ato/types.md]: This is implying that something can operate without ATO indefinitely without an ATO as long as it abides by the following conditions, but "pre-authorization" implies that it is only temporary. Maybe we need a different name for it?

**Comment by afeld** [comment]
Doesn't strike me as controversial, so merging now. We can always follow up with more improvements later.

File: _pages/ato/types.md (hunk 1 above)
Comment (NoahKunin) [inline _pages/ato/types.md]: I don't see how it implies - seems like it says it directly. In the same way, I don't see pre-authorization as implying anything about its duration, but happy to take suggestions on re-naming. I don't think I'm the one who even named it, and if I was, it was offhand in a meeting.

**Pull request merged**